R/auth.R

```diff
@@ -1,112 +1,203 @@
-#' Add authorized user for application
-#' 
-#' @param username The user name.
-#' @param password The password.
-#' @param appDir Directory containing application. Defaults to current working directory.
-#' @examples
-#' \dontrun{
-#' 
-#' # add a user (prompts for password)
-#' addAuthroizedUser("andy")
-#' 
-#' # add a user using supplied password
-#' addAuthorizedUser("tareef", "MrShiny45")
+cleanupPasswordFile <- function(appDir) {
+
+  # normalize appDir path and ensure it exists
+  appDir <- normalizePath(appDir, mustWork = FALSE)
+  if (!file.exists(appDir) || !file.info(appDir)$isdir)
+    stop(appDir, " is not a valid directory", call. = FALSE)
+  
+  # get data dir from appDir
+  dataDir <- file.path(appDir, "shinyapps")
+
+  # get password file
+  passwordFile <- file.path(dataDir, paste("passwords", ".txt", sep=""))
+  
+  # check if password file exists
+  if (file.exists(passwordFile)) {
+    message("WARNING: Password file found! This application is configured to use scrypt ",
+            "authenication, which no longer supported.\nIf you choose to proceed ",
```

Comment:
spelling: authentication

wording: "which *is* no longer supported."

punctuation: "If you choose to proceed, all existing users of this application will be removed and will NOT be recoverable."

wording: "For *more* information please visit"

punctuation: One question mark after proceed not two.